packages/core/src/ceramic.ts

```diff
@@ -244,7 +244,10 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
 
     this._ipfs = modules.ipfs
 
-    this._schemaValidator = new SchemaValidation()
+    const numCores = os.cpus().length
```

Comment:
I had to double check this was logical cores too.. so I think half seems reasonable since it should be cpu bound. I was thinking the full amount of cores might be fine, but I don't know if that would actually help at all or just starve things/context switch too much if both sections are going full bore.